qiita_pet/webserver.py

```diff
@@ -106,6 +104,7 @@ def __init__(self):
             (r"/study/sharing/", ShareStudyAJAX),
             # ORDER FOR /study/description/ SUBPAGES HERE MATTERS.
             # Same reasoning as below. /study/description/(.*) should be last.
+            (r"/study/description/sample_template/", SampleTemplateAJAX),
```

Comment:
They do not. The issue wis we are essentially now maintaining three separate projects in this single repo: qiita core functionality, qiita_pet UI, and the target gene plugin. This is why things are getting a bit odd. The api_proxy folder is what will become the API for qiita core once we separate these projects.